.github/workflows/deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:

```

Comment:
This allows users to trigger a deploy manually from the GitHub UI without pushing a commit